packages/terra-grid/tests/nightwatch/grid-spec.js

```diff
@@ -14,21 +14,18 @@ module.exports = {
   'Displays a default grid': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/grid-tests/default`)
-      .waitForElementPresent('.terra-Grid', 1000)
-      .expect.element('.terra-Grid').to.have.attribute('class').which.contains('terra-Grid');
+      .waitForElementPresent('#grid', 1000);
```

Comment:
@bjankord I remember you saying we want to remove waitForElementPresent and instead use .assert.ElementPresent to speed up test times?